### ISSUE TITLE
Service Error: Using Ghana Mobile Money

### ISSUE THREAD

**Issue by yendiDev** [Flutterwave/flutterwave-flutter#65]
Hi, I want to integrate a mobile money (Ghana) in my app. I use the Flutter plugin (1.0.1) and everything runs well (the UI).

But no payment is processed. When I use the test cards together with all the necessary details, I get "Service Error".

When I use the card payment option, I get "Authentication data required".
If the Flutter plugin for Flutterwave is not ready for production, I would like to know.

**Comment by FredDominant** [comment]
hello @yendiDev , apologies for the inconvenience. This current library will be discontinued. Please use this https://pub.dev/packages/flutterwave_standard

**Comment by yendiDev** [comment]
Can this be used for production in a Flutter application?

**Comment by yendiDev** [comment]
![Screenshot_20210913-150702.jpg](https://user-images.githubusercontent.com/57093800/133109011-518274f8-dd4d-44e2-ab44-f6a4636c8b88.jpg)


I get this error whenever I'm using the mobile money section. The mobile money account is the right one too. What could be the issue?

**Comment by FredDominant** [comment]
Hello @yendiDev, what OTP code did you type in/

**Comment by yendiDev** [comment]
I was sent an OTP, and that was what I used. After I keyed in the OTP, it showed the error message as seen in the screenshot

**Comment by FredDominant** [comment]
For test please use 123456

**Comment by yendiDev** [comment]
I am in live mode

**Comment by yendiDev** [comment]
![Screenshot_20210913-152430.jpg](https://user-images.githubusercontent.com/57093800/133111816-9a57c5b7-10cf-4ccb-9155-8ead4b392f0e.jpg)

**Comment by yendiDev** [comment]
Hello, please any response from the Flutterwave team?

**Comment by FredDominant** [comment]
Hello @yendiDev, this fails because according to the provider, your number is not a valid Mobile Money number.

**Comment by yendiDev** [comment]
Yes, but my number is a valid mobile money number I use on a daily basis. What does Flutterwave define as a valid mobile money number? Is there something I'm missing? @FredDominant

**Comment by listermatrix** [comment]
> Yes, but my number is a valid mobile money number I use on a daily basis. What does Flutterwave define as a valid mobile money number? Is there something I'm missing? @FredDominant

i am facing same issues
